src/metrics/index.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const HttpClient = require("@actions/http-client").HttpClient
+const AmbassadorClient = require('../ambassador/api-client');
+const core = require('@actions/core');
+const configure = require('../configure');
+
+const httpClient = new HttpClient();
+const ambassadorClient = new AmbassadorClient(process.env.TELEPRESENCE_API_KEY)
+
+class MetritonClient {
+    static async sendMetricsReport(action){
+        try {
+            const reportDestination = 'https://metriton.datawire.io/beta/scout'
+            const applicationName = 'telepresence-github-action-integration'
+            const installId = fs.readFileSync(`${configure.getTelepresenceConfigPath()}/id`).toString()
+            core.info('install id: ' + installId)
+            const extensionVersion = '0.0.0-local'
+            const pipelineId = process.env.GITHUB_RUN_ID
```

Comment:
FYI @arturogonzalez58. I think this is an important thing to keep in mind 

> `github.run_id`: A unique number for each workflow run within a repository. This number does not change if you re-run the workflow run.

Reply:
could you give me more context?

Reply:
just wanted to point out that the `run_id` is an identifier that doesn't change if the workflow is re-run in case we were expecting something different on the metriton side when inspecting the activity of the actions.

Reply:
We can mix the run id with the attempt

Reply:
that's a great idea